core/src/main/java/org/jdbi/v3/BuiltInArgumentFactory.java

```diff
@@ -92,6 +99,15 @@
         register(map, Timestamp.class, Types.TIMESTAMP, PreparedStatement::setTimestamp);
         register(map, URL.class, Types.DATALINK, PreparedStatement::setURL);
         register(map, URI.class, Types.VARCHAR, stringifyValue(PreparedStatement::setString));
+
+        Calendar c = Calendar.getInstance();
+        Supplier<Calendar> cal = () -> (Calendar) c.clone();
```

Comment:
`getInstance` is an expensive operation. Clone is cheap. So we pay the `getInstance` penalty once.